app/api/main.py

```diff
@@ -1,9 +1,7 @@
 from fastapi import APIRouter
-
 from app.api.routes import user, login, status, order
 
 api_router = APIRouter()
 api_router.include_router(login.router, prefix="/login", tags=["login"])
 api_router.include_router(user.router, prefix="/users", tags=["users"])
-api_router.include_router(status.router, prefix="/statuses", tags=["statuses"])
```

Comment:
why did you delete the router for statuses?

Reply:
I deleted it by mistake, I'll add it back